snippets/TypescriptReactSetup.md

```diff
@@ -0,0 +1,45 @@
+### TypeScript React Compoent
+
+To get started with React and Typescript you need to import the React module, for Typescript you have to import the whole React Library.
```

Comment:
just write that you need to import the React module. Also write the whole description as one block and put the whole code block below the description. See the 30-seconds-of-code snippets for reference :)